spec/panel-container-element-spec.js

```diff
@@ -19,16 +20,16 @@ describe('PanelContainerElement', () => {
     focus() {}
   }
 
-  const TestPanelContainerItemElement = document.registerElement(
+  window.customElements.define(
     'atom-test-container-item-element',
-    { prototype: TestPanelContainerItemElement_.prototype }
+    TestPanelContainerItemElement_
   );
 
   beforeEach(() => {
     jasmineContent = document.body.querySelector('#jasmine-content');
 
     atom.views.addViewProvider(TestPanelContainerItem, model =>
-      new TestPanelContainerItemElement().initialize(model)
+      new window.customElements.get('atom-test-container-item-element').initialize(model)
```

Comment:
```suggestion
      (new (window.customElements.get('atom-test-container-item-element'))).initialize(model)
```
Explainer:
```javascript
let customElementz = {
  getz (argument) {
    return class example {
      initialize () {return argument}
    }
  }
}
 
new customElementz.getz(15).initialize()
// Uncaught TypeError: customElementz.getz is not a constructor

(new customElementz.getz(15)).initialize()
// Uncaught TypeError: customElementz.getz is not a constructor

new (customElementz.getz(15)).initialize()
// Uncaught TypeError: customElementz.getz(...).initialize is not a constructor

(new (customElementz.getz(15))).initialize()
// 15
```

Reply:
Actual maybe use the `document.createElement` function
```js
document.createElement('atom-test-container-item-element').initialize(model)
```

Tested with
```js
class testElement extends HTMLElement {}

customElements.define('test-element', testElement)

document.createElement('test-element').constructor // class testElement {}
document.createElement('test-element') instanceof testElement // true
```

Reply:
Thanks!